articles/fleetd-authentication.md

```diff
@@ -0,0 +1,101 @@
+
+After starting osquery, `orbit` will also start the `Fleet desktop` tray application. Such application runs as the Desktop GUI user (it does not run as `root`/`SYSTEM`). Before starting `Fleet desktop`, `orbit` will set a random "Fleet desktop token" and send it to Fleet server via the `orbit` authenticated communication channel. `orbit` stores such token on a file accessible to the GUI user and `Fleet desktop` will use the token to authenticate requests to Fleet. When the user clicks on "My device" on the tray icon menu, `Fleet desktop` will open the default browser on a URL that contains the "Fleet desktop token" on its URL path. For this very reason, the "Fleet desktop token" is rotated every hour (by `orbit`).
+
+Here's a diagram to summarize fleetd communication channels after enrollment (more on "tuf.fleetctl.com" below):
+```mermaid
+graph LR;
+
+tuf["tuf.fleetctl.com"];
+
+subgraph fleetd;
+    orbit;
+    osquery;
+    desktop[Fleet desktop];
+end;
+
+server[Fleet];
+
+orbit -- HTTPS w/ orbit node key --> server;
+osquery -- HTTPS w/ osquery node key --> server;
+desktop -- HTTPS w/ Fleet desktop token --> server;
+
+orbit -- HTTPS + The Update Framework --> tuf;
+```
+
+## Fleetd auto updates
+
+`orbit` uses the [The Update Framework](https://theupdateframework.io/) to perform secure auto-updates.
+By default, `orbit` has public root keys embedded in it to only trust updates published and signed in Fleet's TUF repository https://tuf.fleetctl.com (via HTTPS).
+
+fleetd can also be customized to use a user-controlled "TUF" server.
+More information can be found on orbit's Github [README](https://github.com/fleetdm/fleet/tree/main/orbit).
+
+## TLS with custom CA certificates
+
+By default, fleetd uses the OS root CA certificate store and also embeds the [Mozilla CA certificate store](https://curl.se/docs/caextract.html).
+fleetd also supports setting custom root CA certificates that will be used to communicate to Fleet and the TUF server.
+More information can be found on the [Certificates in fleetd]((https://fleetdm.com/guides/certificates-in-fleetd#basic-article) guide.
+
+## mTLS
+
+> The Fleet server does not provide support for mTLS.
+> Users that want to use mTLS on their endpoints must setup a load balancer or TLS terminator like AWS's ELB or nginx that support mTLS.
+
+The fleetd agent does provide basic support for mTLS. See the [Using mTLS](https://fleetdm.com/guides/enroll-hosts#using-mtls) guide.
+
+## ADE and Autopilot MDM flow
+
+When your macOS/Windows endpoints enroll to Fleet via MDM protocols, Fleet will send a command to automatically install fleetd on the enrolled endpoints. Fleet will also send a configuration profile to automatically configure the "enroll secret" and "Fleet URL" so that the enrollment process documented above can take place.
```

Comment:
Each sentence should be on it's own line to make diffs easier.

Reply:
Thanks, I've added new lines.